### PR TITLE
[KOGITO-8304] - Document the usage of all-in-one jobs-service image

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>

**JIRA:**

**Description:**

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] The nav.adoc file has a link to this guide in the proper category
- [ ] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to setup automatic cherry-pick to another branch ?
</summary>

The cherry-pick action allows to setup automatic cherry-pick from `main` to a specific branch.

To allow it, you will need to add the corresponding label with pattern: `backport-{RELEASE_BRANCH}`.  
For example, if a backport to branch `1.26.x` is needed, then the label should be `backport-1.26.x`.

Once the PR is merged, the action will retrieve the commit and cherry-pick it to the desired branch.

*NOTE: You can still add label after the merge and it should still be cherry-picked.*
</details>

### CODE REVIEW

**Review by cristianonicolai — APPROVED** [review]

**Review by radtriste — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc [hunk 1]
```diff
@@ -210,6 +213,13 @@ The example shows how to run PostgreSQL as a Kubernetes deployment, but you can
 ==== Ephemeral
 Alternatively, there is an in-memory database support that does not require any external database configuration, it can be used for testing and development purposes, but it is not recommended for production, since all jobs are lost upon a service restart or failure.
 
+[NOTE]
+====
+The Job Service link:{jobs_service_image_url}[image] is equipped with the PostgreSQL and InMemory options that can be
+switched using the `JOBS_SERVICE_PERSISTENCE`. If not set, it defaults to the `inmemory` option.
+More details about the Job Service Container image can be found link:{jobs_service_image_usage_url}(here).
+====
+
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:222]:
```suggestion
[NOTE]
====
The job service link:{jobs_service_image_url}[image] is equipped with the PostgreSQL and InMemory options that can be switched using the `JOBS_SERVICE_PERSISTENCE`. If not set, it defaults to the `inmemory` option. For more information about the job service, the container image can be found link:{jobs_service_image_usage_url}(here).
====
```

**Review by kaldesai — APPROVED** [review]
Minor suggestions in the note section. Otherwise, your content looks good to me. If you have any questions related to the suggestion please reach out to me. Thank you!

File: serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc [hunk 2]
```diff
@@ -210,6 +213,13 @@ The example shows how to run PostgreSQL as a Kubernetes deployment, but you can
 ==== Ephemeral
 Alternatively, there is an in-memory database support that does not require any external database configuration, it can be used for testing and development purposes, but it is not recommended for production, since all jobs are lost upon a service restart or failure.
 
+[NOTE]
+====
+The Job Service link:{jobs_service_image_url}[image] is equipped with the PostgreSQL and InMemory options that can be
+switched using the `JOBS_SERVICE_PERSISTENCE`. If not set, it defaults to the `inmemory` option.
+More details about the Job Service Container image can be found link:{jobs_service_image_usage_url}(here).
```
Comment (MarianMacik) [inline serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:220]: So I went to the link to see what the other options are and found this:
https://github.com/kiegroup/kogito-images/blob/main/kogito-jobs-service-allinone-overrides.yaml#L25-L27
`The all in one image contains all the supported jdbc plugins for Jobs Service, ephemeral, infinispan, mongodb and infinispan, if empty, defaults to ephemeral`
So the default is ephemeral according to the description and there are also infinispan and mongodb options to choose from.

Comment (spolti) [reply serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:220]: correct, but, AFAIK for serverless only postgresql and inmemory are supported at this time

Comment (MarianMacik) [reply serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:220]: That may be true but we still say it defaults to `inmemory` whereas in the page it says it defaults to `ephemeral`.

File: serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc [hunk 3]
```diff
@@ -210,6 +213,16 @@ The example shows how to run PostgreSQL as a Kubernetes deployment, but you can
 ==== Ephemeral
 Alternatively, there is an in-memory database support that does not require any external database configuration, it can be used for testing and development purposes, but it is not recommended for production, since all jobs are lost upon a service restart or failure.
 
+[NOTE]
+====
```
Comment (spolti) [inline serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:217]:
```suggestion
```

File: serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc [hunk 4]
```diff
@@ -210,6 +213,16 @@ The example shows how to run PostgreSQL as a Kubernetes deployment, but you can
...
+The Job Service link:{jobs_service_image_url}[image] is equipped with the PostgreSQL and InMemory options that can be
```
Comment (spolti) [inline serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:218]:
```suggestion
```

File: serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc [hunk 5]
```diff
@@ -210,6 +213,16 @@ The example shows how to run PostgreSQL as a Kubernetes deployment, but you can
...
+switched using the `JOBS_SERVICE_PERSISTENCE`. If not set, it defaults to the `inmemory` option.
```
Comment (spolti) [inline serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:219]:
```suggestion
```

File: serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc [hunk 6]
```diff
@@ -210,6 +213,16 @@ The example shows how to run PostgreSQL as a Kubernetes deployment, but you can
...
+More details about the Job Service Container image can be found link:{jobs_service_image_usage_url}(here).
```
Comment (spolti) [inline serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:220]:
```suggestion
```

File: serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc [hunk 7]
```diff
@@ -210,6 +213,16 @@ The example shows how to run PostgreSQL as a Kubernetes deployment, but you can
...
+====
+[NOTE]
+====
+The job service link:{jobs_service_image_url}[image] is equipped with the PostgreSQL and InMemory options that can be switched using the `JOBS_SERVICE_PERSISTENCE`. If not set, it defaults to the `inmemory` option. For more information about the job service, the container image can be found link:{jobs_service_image_usage_url}(here).
```
Comment (spolti) [inline serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:224]:
```suggestion
The job service link:{jobs_service_image_url}[image] is equipped with the PostgreSQL and Ephemeral (InMemory) options that can be switched using the `JOBS_SERVICE_PERSISTENCE`. If not set, it defaults to the `inmemory` option. For more information about the job service, the container image can be found link:{jobs_service_image_usage_url}(here).
```

File: serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc [hunk 8]
```diff
@@ -210,6 +213,11 @@ The example shows how to run PostgreSQL as a Kubernetes deployment, but you can
 ==== Ephemeral
 Alternatively, there is an in-memory database support that does not require any external database configuration, it can be used for testing and development purposes, but it is not recommended for production, since all jobs are lost upon a service restart or failure.
 
+====
+[NOTE]
+====
+The job service link:{jobs_service_image_url}[image] is equipped with the PostgreSQL and Ephemeral (InMemory) options that can be switched using the `JOBS_SERVICE_PERSISTENCE`. If not set, it defaults to the `inmemory` option. For more information about the job service, the container image can be found link:{jobs_service_image_usage_url}(here).
```
Comment (MarianMacik) [inline serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc:219]: I believe it should be
```suggestion
The job service link:{jobs_service_image_url}[image] is equipped with the PostgreSQL and Ephemeral (InMemory) options that can be switched using the `JOBS_SERVICE_PERSISTENCE`. If not set, it defaults to the `ephemeral` option. For more information about the job service, the container image can be found link:{jobs_service_image_usage_url}(here).
```
if we want to use terms used by the image itself.

**Review by ricardozanini — APPROVED** [review]

**Pull request merged**